x/tx/textual/internal/testdata/decimals.json

```diff
@@ -1,9 +1,9 @@
 [
-  ["0", "0"],
-  ["1", "1"],
-  ["12", "12"],
-  ["123", "123"],
-  ["1234", "1'234"],
+  ["0.0", "0"],
+  ["1.0", "1"],
```

Comment:
I remember when we introduced gov v1, we dropped the customtype to sdk.Dec, see [here](https://github.com/cosmos/cosmos-sdk/blob/21cfde0f3b09799d607e14cde9dcda331beec8e7/proto/cosmos/gov/v1/gov.proto#L35) or tally. This means in state, the string is stored as a "normal" decimal.

What we need to make sure is that the string does always have a `.`. For example, we should test this manually with Ledger, using a weighted vote of `1`, and make sure it shows `1` and not `1e-18`.

Reply:
Will test that 👌

Reply:
```
...Options (1/1)xWeightedVoteOption object�fOptionoVOTE_OPTION_YES�fWeighta1�nEnd of Options�nEnd of Message�iGas limitg200'000...
```
![WhatsApp Image 2023-02-23 at 3 17 02 PM](https://user-images.githubusercontent.com/14063057/220995571-960da3ec-4da1-4a51-a248-f5177d1456fe.jpeg)

Looks fine 👌 Also for bank send it looks good.

Reply:
```
...Options (1/1)xWeightedVoteOption object�fOptionoVOTE_OPTION_YES�fWeighta1�nEnd of Options�nEnd of Message�iGas limitg200'000...
```
![WhatsApp Image 2023-02-23 at 3 17 02 PM](https://user-images.githubusercontent.com/14063057/220995571-960da3ec-4da1-4a51-a248-f5177d1456fe.jpeg)

Looks fine 👌 Also for bank send it looks good.